meetings/2017-01-26.md

```diff
@@ -0,0 +1,203 @@
+
+Discussion Group advised\* not accepting `nvm` till further decisions regarding
+standards and specs such runtime managers should adhere to.
+
+\*See https://github.com/nodejs/TSC/issues/96#issuecomment-264965326 and bottom
+of
+https://github.com/nodejs/version-management/blob/master/meetings/2016-12-01.md.
+
+We should make sure this conclusion is clear in the thread and perhaps close it.
+`nvm` could re-apply later.
+
+There are benefits of having a version or runtime manager so we should leave the
+repo and Discussion Group active.
+
+**Next steps**
+
+* Update #96 and remove from tsc-agenda, but leave repo active.
+
+---
+
+### Copyright
+
+* Updating the Copyright
+  [TSC#174](https://github.com/nodejs/TSC/issues/174)
+* Copyright Date
+  [TSC#195](https://github.com/nodejs/TSC/issues/195)
+
+We would benefit from more feedback from the Foundation Legal Committee. Can we
+set up a process to request a review? Could be partly in private.
+
+Once we solve pending issues we can ask for more review of our processes like
+CLAs. More process can be an impediment to contributors, though.
+
+Need to also ensure compliance for adjacent projects like node-inspect and
+node-report in addition to core.
+
+@mikeal: Board made a list of requirements for bringing in new projects, for
+example acceptable licenses and DCO.
+
+Should add this list of requirements to TSC repo for future reference,
+@williamkapke to prepare PR.
+
+Some reliable sources have told us that a date isn’t required. But if a date was
+previously included it can’t be removed.
+
+**Next steps**
+
+* PR to add list of requirements for new projects to TSC repo.
+
+---
+
+### Governance & TSC
+
+* charter: revised membership rules
+  [TSC#142](https://github.com/nodejs/TSC/pull/142)
+
+Ping @jasnell for update.
+
+---
+
+* General update to the Member travel fund doc
+  [TSC#181](https://github.com/nodejs/TSC/pull/181) 
+
+@mikeal recommended mentioning that traveler should update sheet with actual
+costs after trip. @addaleax replied that it might not be known to the traveler.
+@mikeal replied today that NF member (e.g. Mikeal) can provide final numbers.
+
+Point is we need to track how much we’ve spent so we can request budget
+adjustments if needed.
+
+Should also add to document how much money has been allocated and spent.
+
+@williamkapke suggested merging as is and addressing concerns in later PRs.
+
+**Next steps**
+
+@Fishrock123 will continue discussion and help merge.
+
+---
+
+* Rotating meeting times
+  [TSC#198](https://github.com/nodejs/TSC/issues/198)
+
+Is the solution to change the time rather than to rotate them?
+
+Are we preventing interested parties from joining?
+
+We need to clarify if some interested parties are unable to attend; we should
+only adjust to accommodate actual participants.
+
+Goal is always to reach consensus in GitHub if possible, meetings are a
+fallback.
+
+Can we find a time which works for Ben, Rod, and American folks?
```

Comment:
Nice to see this was discussed.  0500 or 1600 UTC would work better for me but probably isn't very nice to people from the USA (and @rvagg?  I don't know what timezone he's in after moving.)